vtStorManaged/vtStorManaged.vcxproj

```diff
@@ -43,6 +43,7 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|x64'" Label="Configuration">
     <PlatformToolset>v120</PlatformToolset>
     <ConfigurationType>DynamicLibrary</ConfigurationType>
+    <UseDebugLibraries>true</UseDebugLibraries>
```

Comment:
Add missing config to declare debug mode support "UseDebugLibraries"
Default UseDebugLibraries is false that means no using debug libraries